Source/WebKit/UIProcess/Cocoa/GPUProcessProxyCocoa.mm

```diff
@@ -39,6 +44,30 @@
     parameters.applicationVisibleName = applicationVisibleName();
 }
 
+#if HAVE(POWERLOG_TASK_MODE_QUERY)
+bool GPUProcessProxy::powerLoggingInTaskMode() const
```

Comment:
I'd suggest calling this isPowerLoggingInTaskMode(). It tends to read better at call sites when functions that return boolean states start with "is" or "should".